src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -86,14 +90,18 @@ export function PostThreadItem({
   hideTopBorder?: boolean
 }) {
   const postShadowed = usePostShadow(post)
+
+  const [translatedText, setTranslatedText] = useState<string>(record.text)
```

Comment:
not sure if you prefer a different approach here other than useState and prop drilling